stubs/PDOStatement.stub

```diff
@@ -1,8 +1,9 @@
 <?php
 
 /**
- * @implements Traversable<array<int|string, mixed>>
- * @implements IteratorAggregate<array<int|string, mixed>>
+ * @template RowType of array
```

Comment:
Why is this `of array` but not in Psalm? https://github.com/vimeo/psalm/blob/93e9054f98b040bf246c2023201503db3dbf162c/stubs/pdo.phpstub#L4 I want to merge it only if they're exactly the same, minus the `taint-sink` stuff.